CMakeLists.txt

```diff
@@ -365,7 +365,7 @@ else(BISON_FOUND)
       " and retry running cmake."
     )
   endif()
-endif(NOT BISON_FOUND)
+endif(BISON_FOUND)
```

Comment:
is this change intended?

Reply:
Yes, it is intended. cmake throws a warning since a logical block closes with mis-matching arguments